File: api/Models/Room.cs
using System.Collections.Generic;

namespace iTechArt.Hotels.Api.Models
{
    public class Room
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int Sleeps { get; set; }
        public int Number { get; set; }
        public decimal Price { get; set; }
        public int? MainImageId { get; set; }
        public List<Facility> Facilities { get; set; } = new List<Facility>();
    }
}

File: admin/src/app/Dtos/room.ts
import { Facility } from './facility';
import { Hotel } from './hotel';

export interface Room {
  id: number;
  name: string;
  sleeps: number;
  price: number;
  facilities: Facility[];
  mainImageId?: number;
}

export type RoomToReserve = Omit<Hotel, 'id'>;
